### DISCUSSION TITLE
Starting discussion as external GH member

### DISCUSSION THREAD

**Discussion by rantoniuk** [community/community#3124] [Discussions]
How can one grant permission for people not being part of the team/organisation to start new discussions in a specific category (e.g. Q&A)? I couldn't find any documentation about the permissions configuration..

**Comment by evi-liu** [comment]
Do you mean external members who don't have GitHub accounts at all? If it's a public repository and the user is logged in, anyone should be able to start a new discussion.

**Reply by rantoniuk** [reply]
No, I mean a logged in GH user to start a new discussion in an organisation's repository. 
Try to start a new discussion [in our organisation's repository](https://github.com/jenkinsci/jira-plugin/discussions) - the repository is public but there is no button and I'm guessing it is blocked somehow on organisation level.

**Reply by evi-liu** [reply]
Ah, yes. In your organization settings, there should be a setting for specifying which permissions can create a new discussion.

**Reply by rantoniuk** [reply]
In the organisation settings I can only see:
"Enable team discussions for this organization. This allows members to start discussions in any team in the CloudFolks organization."

But I am asking about enabling to start discussion for *non-members* of the organisation.

**Reply by rantoniuk** [reply]
@evi-liu do you have any suggestions?

**Reply by rantoniuk** [reply]
Oh, I now see a new option in the organisation settings:
_If enabled, all users with read access can create and comment on discussions in CloudFolks’s repositories.
If disabled, discussion creation is limited to users with at least triage permission. Users with read access can still comment on discussions._

I guess this is what was needed.
